app/routes/login/index.tsx

```diff
@@ -0,0 +1,45 @@
+import { Links, LiveReload, Meta, Scripts } from "@remix-run/react";
+import { LoaderFunction, MetaFunction, redirect } from "@remix-run/node";
+import { CriticalCss } from "@webstudio-is/sdk";
+import { darkTheme } from "~/shared/design-system";
+
+import { authenticator } from "~/services/auth.server";
+import config from "~/config";
+
+import { Login, links } from "~/auth/";
+
+export { links };
+
+export const meta: MetaFunction = () => {
+  return { title: "Webstudio Login" };
+};
+
+const LoginRoute = () => {
```

Comment:
I think we should reuse this document between designer

Reply:
What do you mean?

Reply:
the html document you are sending here, is a copy/paste, we should reuse it, don't see differences

Reply:
Where should it go?